### ISSUE TITLE
Could not load actor and crew information  

### ISSUE THREAD

**Issue by luke0825** [jellyfin/jellyfin#2640]
**Describe the bug**

Actor and crew information often fails to load, sometimes it works.

**System (please complete the following information):**
 - OS: [Ubuntu 18.04]
 - Virtualization: [none]
 - Clients: [Browser and Android]
 - Browser: [Chrome 80]
 - Jellyfin Version: [10.5.0]
 - Playback: [Direct Play] 
 - Installed Plugins: [default]
 - Reverse Proxy: [none]
 - Base URL: [none]
 - Networking: [host]
 - Storage: [local]

**To Reproduce**

1. Go to any film
2. Click on Cast & Crew
3. It's always loading

**Logs**

A few minutes later the following error was reported：
[2020-03-22 12:22:41.429 +08:00] [ERR] Error processing request: "The operation was canceled."
[2020-03-22 12:22:41.431 +08:00] [WRN] HTTP Response 500 to "192.168.50.134". Time (slow): 0:01:40.0204367. "http://192.168.50.200:8096/Users/8e7fcc6580e64dcba707092bd1246f84/Items/c600f7c332eb92d4a07e228cc52fcece"

**Screenshots**

![20200322130153](https://user-images.githubusercontent.com/57798606/77242805-847fe300-6c3d-11ea-89ed-d7b9953d362e.jpg)
![20200322130255](https://user-images.githubusercontent.com/57798606/77242807-88136a00-6c3d-11ea-9e5f-9378feca6aa5.jpg)

**Comment by dga4130-renato** [comment]
i had that bug also, seems because it doesnt fetch the right info.
actors missing almost was the problem.. and i had to select on each actor reload metadata, next actor description and photo was loading. just deleted the library.db and scanned all over again, now works all fine also the tmdb plugin and the collection option wasnt working right , after that all fine.
seems someting in the library settings, like fetch from internet or so that i had disabled from the first installation as i had infos in each movie subdir generated by tmm.

**Comment by marcushsu** [comment]
I run into the same problem. The problem cannot be solved even after deleting the library.db and rescanning the library as suggested above. I have two jellyfin systems and both of them are the latest docker image version 10.5.5, pulled from the official docker hub. The one on Synology DS418play has this problem but the one on CentOS 7 does the function well.

**Comment by vizhen** [comment]
I have the same problem. After I modified the moviedb api hosts.  It works well. My jellyfin version is 10.5.5 and OS: OMV

**Comment by Deepcuts** [comment]
> I have the same problem. The problem cannot be solved even after deleting the library.db and rescanning the library as suggested above. Then I modified the moviedb api hosts. It works well. My jellyfin version is 10.5.5 and OS: OMV.

Maybe share your fix?

**Comment by HansUrst** [comment]
I run in the same issue. Since like a month it has not worked. Could you please open this issue?

**Comment by ouafouaf** [comment]
I had the same issue, actors pages were loading and loading but never going anywhere.

The logs showed:
`Jellyfin.Server.Middleware.ResponseTimeMiddleware: Slow HTTP Response from.......`
and 
`MediaBrowser.Providers.People.PersonMetadataService: Error in "TheMovieDb" `
`System.Net.Http.HttpRequestException: Cannot assign requested address (api.themoviedb.org:443)`

I tried to remove themoviedb from the metadata configuration on an existing library, but didnt work.
I did a fresh install of jellyfin, started a new movie library, removed themoviedb from the library metadata config, and now it works fine.   

Probably just an issue connecting to themoviedb from China.
